### PR TITLE
Updating the view and CSS, no pic and twitter handle in each tweet;

### PR DESCRIPTION
**Pull request by bl00dymarie**
Displaying only 3 most tweets

### CODE REVIEW

**Comment by bl00dymarie** [comment]
Hey @theaamanda hier ist der Code bzgl. CSS update. Kannst das durchschauen und dann mergen!
<3

File: app/assets/stylesheets/twitter.scss [hunk 1]
```diff
@@ -5,30 +5,14 @@
 .twitter_header {
   font-size: 20px;
   text-align: left;
-  margin-bottom: 30px;
+  margin-bottom: 80px;
```
Comment (bl00dymarie) [inline app/assets/stylesheets/twitter.scss:8]: adding space for twitter avatar pic and twitter handle to display

File: app/helpers/twitter_helper.rb [hunk 2]
```diff
@@ -18,6 +18,6 @@ def twitter_name
   end
 
   def tweets
-    twitter_client.user_timeline(twitter_handle).select{ |tweet| tweet.text.start_with?('RT','@')==false}
+    twitter_client.user_timeline(twitter_handle).select{ |tweet| tweet.text.start_with?('RT','@')==false}.take(3)
```
Comment (bl00dymarie) [inline app/helpers/twitter_helper.rb:21]: Displays 3 most recent tweets, still without RT or @responses

File: app/views/home/_content.html.haml [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
-  .col-md-9
+  .row
     = home_page_buttons
-  .col-md-3
+
+.col-md-3
+  .row
     = render 'twitter/index'
 
     - if Rails.env == 'development'
```
Comment (bl00dymarie) [inline app/views/home/_content.html.haml:13]: Dividing the whole view into 2 columns, incl. "Welcome" on the web app

**Comment by froilainhaeckse** [comment]
Things we should cover in future from my point of view:
- change default-picture
- check alignment and align correctly
- decreasing font sizes, at least for header
- delete or modify footer text

**Pull request merged**